tests/unit/common/eeprom/msgpack.h

```diff
@@ -0,0 +1,1020 @@
+//
+// Created by Mike Loomis on 6/22/2019.
```

Comment:
Should this go somewhere into lib/third_party/...? Should it be a git subproject, to allow future easy updates?

Reply:
Maybe,I am thinking about ditching it and using something ours to cut down usage of eeprom and get rid of dynamic allocation

Reply:
That's also a possibility. We could have something simple/simplish. Another option is looking at how protobuf does it (I mean only the lower-level thing without the generators, etc, but they somehow combine the „name“ tag with the „type“ tag). But on the other hand, they have some variable-length thingie, so that's maybe a complication.